GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/WorkingDirectoryTests.cs

```diff
@@ -345,23 +345,28 @@ public void HydratingNestedFileUsesNameCaseFromRepo()
         }
 
         [TestCase, Order(9)]
-        public void WriteToHydratedFileAfterRemount()
+        public void AppendToHydratedFileAfterRemount()

```

Comment:
I decided to use an existing test to verify the ">>".
I can also create a new one.
My thought process was, this does the work already and let's not increase the time of the functional tests.  Let me know if you the tradeoffs aren't fair here.

Reply:
I think expanding the current test looks good.  To confirm, this test fails without your fix?

Reply:
I think this is good for this test but I would also like a test in `StatusTests` so that we are covering this case with `git status` as well.

Reply:
Added a status test for good measure